src/transform/FooterReadMore.js

```diff
@@ -0,0 +1,336 @@
+import './FooterReadMore.css'
+
+/**
+ * @typedef {function} SaveButtonClickHandler
+ * @param {!string} title
+ * @return {void}
+ */
+
+/**
+ * @typedef {function} TitlesShownHandler
+ * @param {!Array.<string>} titles
+ * @return {void}
+ */
+
+/**
+ * Display fetched read more pages.
+ * @typedef {function} ShownReadMorePagesHandler
+ * @param {!Array.<object>} pages
+ * @param {!string} containerID
+ * @param {SaveButtonClickHandler} saveButtonClickHandler
+ * @param {TitlesShownHandler} titlesShownHandler
+ * @param {!Document} document
+ * @return {void}
+ */
+
+const _saveButtonIDPrefix = 'readmore:save:'
```

Comment:
This looks like a static immutable constant, would you mind if we stuck with `UPPER_CASE`-like syntax for these. So, SAVE_BUTTON_ID_PREFIX? I guess our functions are also constants but this is constant data.